example/particle_tracers/particle_tracers.cpp

```diff
@@ -397,6 +400,7 @@ void ProblemGenerator(MeshBlock *pmb, ParameterInput *pin) {
 
         y(n) = y_min + rng_gen.drand() * (y_max - y_min);
         z(n) = z_min + rng_gen.drand() * (z_max - z_min);
+        id(n) = (num_tracers / nbtotal) * gid + n;
```

Comment:
should this actually be `id(n) = num_tracers_meshblock * gid + n;`?

Reply:
No---doing that produces overlapping ids. I didn't quite track down what that is, though. I just wanted something sufficiently sparse that no ids overlapped.

Reply:
I think `num_tracers_meshblock` is not computed as the total tracer number divided by meshblock number. It's computed by integrated the advected "mass." But that means that since it varies per meshblock it's not useful for a total "ID" space.

Reply:
Oh ok, I must have misunderstood `num_tracers_meshblock`. Does the `id(n) = ` line as is guarantee no overlaps though? Isn't the stride even per meshblock but what if an earlier meshblock has many more particles than the length of that "average" stride?

Reply:
For simplicity you could probably just do `num_tracers * gid + n` right?

Reply:
I am not sure. I didn't really dig in too much. For sure, we could just do `num_tracers * gid + n`. That's guaranteed to work.

Reply:
done